languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/added.txt

```diff
@@ -8320,4 +8320,141 @@ Schaltwippen	Schaltwippe	SUB:NOM:PLU:FEM
 Schaltwippe	Schaltwippe	SUB:AKK:SIN:FEM
 Schaltwippe	Schaltwippe	SUB:DAT:SIN:FEM
 Schaltwippe	Schaltwippe	SUB:GEN:SIN:FEM
-Schaltwippe	Schaltwippe	SUB:NOM:SIN:FEM
\ No newline at end of file
+Schaltwippe	Schaltwippe	SUB:NOM:SIN:FEM
+Reiher	Reiher	SUB:NOM:SIN:MAS
+Reihers	Reiher	SUB:GEN:SIN:MAS
+Reiher	Reiher	SUB:DAT:SIN:MAS
+Reiher	Reiher	SUB:AKK:SIN:MAS
+Reiher	Reiher	SUB:NOM:PLU:MAS
+Reiher	Reiher	SUB:GEN:PLU:MAS
+Reihern	Reiher	SUB:DAT:PLU:MAS
+Reiher	Reiher	SUB:AKK:PLU:MAS
+Anime	Anime	SUB:NOM:SIN:MAS
+Animes	Anime	SUB:GEN:SIN:MAS
+Anime	Anime	SUB:DAT:SIN:MAS
+Anime	Anime	SUB:AKK:SIN:MAS
+Animes	Anime	SUB:NOM:PLU:MAS
+Animes	Anime	SUB:GEN:PLU:MAS
+Animes	Anime	SUB:DAT:PLU:MAS
+Animes	Anime	SUB:AKK:PLU:MAS
+Date	Date	SUB:NOM:SIN:NEU
+Dates	Date	SUB:GEN:SIN:NEU
+Date	Date	SUB:DAT:SIN:NEU
+Date	Date	SUB:AKK:SIN:NEU
+Dates	Date	SUB:NOM:PLU:NEU
+Dates	Date	SUB:GEN:PLU:NEU
+Dates	Date	SUB:DAT:PLU:NEU
+Dates	Date	SUB:AKK:PLU:NEU
+Dingsbums	Dingsbums	SUB:NOM:SIN:MAS
+Dingsbums	Dingsbums	SUB:GEN:SIN:MAS
+Dingsbums	Dingsbums	SUB:DAT:SIN:MAS
+Dingsbums	Dingsbums	SUB:AKK:SIN:MAS
+Dingsbums	Dingsbums	SUB:NOM:PLU:MAS
+Dingsbums	Dingsbums	SUB:GEN:PLU:MAS
+Dingsbums	Dingsbums	SUB:DAT:PLU:MAS
+Dingsbums	Dingsbums	SUB:AKK:PLU:MAS
+Dingsbums	Dingsbums	SUB:NOM:SIN:NEU
+Dingsbums	Dingsbums	SUB:GEN:SIN:NEU
+Dingsbums	Dingsbums	SUB:DAT:SIN:NEU
+Dingsbums	Dingsbums	SUB:AKK:SIN:NEU
+Dingsbums	Dingsbums	SUB:NOM:PLU:NEU
+Dingsbums	Dingsbums	SUB:GEN:PLU:NEU
+Dingsbums	Dingsbums	SUB:DAT:PLU:NEU
+Dingsbums	Dingsbums	SUB:AKK:PLU:NEU
+Dingsbums	Dingsbums	SUB:NOM:SIN:FEM
+Dingsbums	Dingsbums	SUB:GEN:SIN:FEM
+Dingsbums	Dingsbums	SUB:DAT:SIN:FEM
+Dingsbums	Dingsbums	SUB:AKK:SIN:FEM
+Dingsbums	Dingsbums	SUB:NOM:PLU:FEM
+Dingsbums	Dingsbums	SUB:GEN:PLU:FEM
+Dingsbums	Dingsbums	SUB:DAT:PLU:FEM
+Dingsbums	Dingsbums	SUB:AKK:PLU:FEM
+Flips	Flips	SUB:NOM:SIN:MAS
```

Comment:
Hier sollte `NOG` benutzt werden, siehe https://morphy.wolfganglezius.de/content/2-download/wklassen.pdf

Reply:
Mein Kommentar bezog sich auf "Flips", nicht auf Dingsbums. "Flips" scheint ja nur in der Mehrzahl aufzutreten.